proposals/0303-swiftpm-extensible-build-tools.md

```diff
@@ -15,7 +15,7 @@ This is a proposal for extensible build tools support in Swift Package Manager.
 
```

Comment:
add link to "previous version" in the top section

Reply:
Will do.  Thanks for pointing that out.